runtime/perfetto/db.go

```diff
@@ -95,16 +95,22 @@ type DB struct {
 	replicaNumCache *lru.Cache[replicaCacheKey, int]
 }
 
+// DatabaseFilePath returns the path of the file that stores the trace database.
+func DatabaseFilePath() (string, error) {
+	dataDir, err := files.DefaultDataDir()
+	if err != nil {
+		return "", err
+	}
+	// TODO(mwhittaker): Use a different db for every deployer.
```

Comment:
Let me know if you want to tackle this in a future PR! Right now, `weaver single purge` deletes the perfetto db used by `weaver multi` (and vice versa), which is a little odd. I'm happy to work on it too, but I'll give you first dibs :)

Reply:
Done.  Although I am not happy with the strings constants I sprinkled around.  It would've helped if there was a type for specifying _mode_ of weaver tool but I didn't want to make that change in context of this CL.  Let me know if you have any suggestions.